src/styles/main.scss

```diff
@@ -1,3 +1,83 @@
+@import "./variables.scss";
+
 body {
   margin: 0;
 }
+
+* {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.calendar {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: flex-start;
+  gap: 1px;
+  margin-top: 100px;
+  margin-left: $general-margin;
+  margin-right: $general-margin;
```

Comment:
You can use shorthand property
https://developer.mozilla.org/en-US/docs/Web/CSS/margin#syntax